internal/bundle/bundles/deploy_then_remove_resources/databricks_template_schema.json

```diff
@@ -2,7 +2,7 @@
     "properties": {
         "unique_id": {
             "type": "string",
-            "description": "Unique ID for job name"
+            "description": "Unique ID for pipeline name"
```

Comment:
What happened here?

Reply:
Fixing an very minor error from before. The resource in this text fixture template is a pipeline.